src/open_company/api/entry.clj

```diff
@@ -0,0 +1,50 @@
+(ns open-company.api.entry
+  (:require [compojure.core :refer (defroutes GET OPTIONS)]
+            [liberator.core :refer (defresource by-method)]
+            [open-company.api.common :as common]
+            [open-company.resources.company :as company-res]
+            [open-company.representations.common :as common-rep]
+            [open-company.representations.company :as company-rep]
+            [cheshire.core :as json]))
+
+;; ----- Representations
+;; I assume these could go into a separate representations ns but since
+;; they're just links and not full domain objects I just kept them here
+
+(defn company-link [{:keys [name] :as c}]
```

Comment:
I think I'd rather have them in the representation ns. I agree it doesn't warrant it's own namespace, but let's put them in representations.common